en/contributing/README.md

```diff
@@ -50,14 +49,20 @@ Now you are all set: You have a directory `JabRef` containing the recent updates
 
 Try a branch:
 
-1. `cd` into the `JabRef` directory: `cd c:\git-repositories\JabRef`
-2. Checkout out the pull request: `gh pr checkout 13111` - where `13111` is the PR number, in this case [pr#13111](https://github.com/JabRef/jabref/pull/13111).
-3. Compile and run JabRef: `gg.cmd gradle run :jabgui:run` (on Linux and macOS, you need to prefix it with `sh`: `gg.cmd gradle run :jabgui:run`). This will also download the necessary JDK and gradle distribution. On the first run, please give the system enough time to accommodate and wait until the JabRef window launches. Depending on your hardware, this may take minutes.
+1. `cd` into the `jabref` directory: `cd c:\git-repositories\jabref`
+2. Checkout out the pull request: `sh ./gg.cmd gh pr checkout 13111` - where `13111` is the PR number, in this case [pr#13111](https://github.com/JabRef/jabref/pull/13111).
```

Comment:
gh pr checkout also works with the fork and branch name, e..g  you can simply click on that copy button in the titlte 

![grafik](https://github.com/user-attachments/assets/0682e443-00e7-4b59-a639-9680251524c9)

`Yubo-Cao:walkthrough `